packages/api/cms-api/schema.gql

```diff
@@ -539,6 +541,8 @@ type Mutation {
   createPageTreeNode(input: PageTreeNodeCreateInput!, scope: PageTreeNodeScopeInput!, category: String!): PageTreeNode!
   triggerKubernetesCronJob(name: String!): KubernetesJob!
   currentUserSignOut: String!
+  userPermissionsImpersonate(userId: String!): Boolean!
+  userPermissionsStopImpersonate: Boolean!
```

Comment:
What do you think about userPermissionsStartImpersonation and userPermissionsStopImpersonation?